lxc/import.go

```diff
@@ -33,6 +34,7 @@ func (c *cmdImport) Command() *cobra.Command {
 
 	cmd.RunE = c.Run
 	cmd.Flags().StringVarP(&c.flagStorage, "storage", "s", "", i18n.G("Storage pool name")+"``")
+	cmd.Flags().StringVarP(&c.flagName, "name", "n", "", i18n.G("Instance name")+"``")
```

Comment:
I thought we said we'd just make it an optional argument (not a flag)?

Reply:
Ah right, ok I misunderstood. Will change.